### PR TITLE
Release v0.9.2

### PR DESCRIPTION
**Pull request by jwoertink**
This just fixes an issue with specs hanging when using the latest chromedriver.

@mdwagner  I also changed the version structure to match what you were doing on the other shards. Let me know if this looks ok.

### CODE REVIEW

**Review by mdwagner — APPROVED** [review]
Yes, this is the pattern that I'm trying to follow for the other shards, but just to verify we have the right lower bounds for our dependencies. Ideally, I think if we _can_ use a base minor version, we probably should. Another thing, we can use Crystal macros to backport upgrades as well.

File: shard.yml [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-crystal: ">= 1.4.0"
+crystal: "~> 1.4"
 
 license: MIT
 
 dependencies:
   selenium:
     github: crystal-loot/selenium.cr
-    version: ~> 0.12
+    version: ">= 0.12.1, < 0.13"
```
Comment (mdwagner) [inline shard.yml:14]: Looks good, but just want to make sure that `0.12.1` is actually _required_ instead of `0.12.1`?

File: shard.yml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
   webdrivers:
     github: crystal-loot/webdrivers.cr
-    version: ~> 0.4
+    version: ">= 0.4.3, < 0.5"
```
Comment (mdwagner) [inline shard.yml:17]: Same here, just want to verify

File: shard.yml (hunk 1 above)
Comment (jwoertink) [reply shard.yml:14]: Yeah, 0.12.0 has issues, so we need 0.12.1 at a minimum for this to work.

File: shard.yml (hunk 2 above)
Comment (jwoertink) [reply shard.yml:17]: Same here. It needs to be at least 0.4.3 to work. 0.4.2 doesn't

File: shard.yml [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
   habitat:
     github: luckyframework/habitat
-    version: ~> 0.4.7
+    version: ">= 0.4.7, < 0.5"
```
Comment (mdwagner) [inline shard.yml:20]: I'm fine with this one, because we were already requiring it to be `0.4.7`, whereas the other ones were not.

**Comment by mdwagner** [comment]
LGTM

**Pull request merged**